src/javascripts/Services/referencedByViewService.js

```diff
@@ -0,0 +1,43 @@
+ngapp.service('referencedByViewService', function($timeout, layoutService, settingsService, xelibService, referencedByViewFactory, objectUtils) {
+    this.buildFunctions = function(scope) {
+        // inherited functions
+        scope.releaseTree = referencedByViewFactory.releaseTree;
+
+        scope.buildTree = function() {
+            xelib.SetSortMode(scope.sort.column, scope.sort.reverse);
+            scope.tree = xelib.GetReferencedBy(scope.record).map(function(handle) {
+                return scope.buildNode(handle);
+            });
+        };
+
+        // PUBLIC
+        scope.onDragOver = function() {
```

Comment:
if we end up linking this to the record view we won't need drag and drop

Reply:
(well, we could keep drag and drop so the referenced by view can still display records even if it isn't linked to a record view)  :thinking: